plugins/USBPrinting/USBPrinterOutputDevice.py

```diff
@@ -119,51 +89,52 @@ def __init__(self, serial_port, parent = None):
         self._control_view = None
 
     onError = pyqtSignal()
-    progressChanged = pyqtSignal()
-    extruderTemperatureChanged = pyqtSignal()
-    bedTemperatureChanged = pyqtSignal()
+
     firmwareUpdateComplete = pyqtSignal()
 
     endstopStateChanged = pyqtSignal(str ,bool, arguments = ["key","state"])
 
-    @pyqtProperty(float, notify = progressChanged)
-    def progress(self):
-        return self._progress
+    def _setTargetBedTemperature(self, temperature):
+        Logger.log("d", "Setting bed temperature to %s", temperature)
```

Comment:
It might be useful to also log for which device this is happening, in case you have multiple devices.